make.config.in

```diff
@@ -58,7 +58,9 @@ FACETS_SOURCE   = @FACETS_SOURCE@
 # Files that are needed by configure and should be dependencies for 'all'
 
 OBJ						= $(SOURCEC:%.cxx=%.o)
+ifndef RELEASE
```

Comment:
This works the other way.
If `RELEASE` isn't set the process works as it did so far.
Unfortunately I did not make a comment as to why it is required that LIB isn't set in the release build ...
I think it might be related to make sure BOUT++ is not recompiled after it is installed.
The variable should probably be called `RELEASED` instead of `RELEASE` - as it is set if the `make.config` is installed.
Preventing the recompilation is the reason for the change in line [85](https://github.com/boutproject/BOUT-dev/pull/550/files#diff-4a17407e6513b8ab69ae8e1c2186de69R85)

If you are worried that this cause issues if `RELEASE` is defined as an environment variable, another name could be chosen (e.g. `BOUT_IS_RELEASED`)